<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSuperPasswordClass</class>
 <widget class="QDialog" name="QSuperPasswordClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>315</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>315</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SuperCode 2.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QSuperPassword.qrc">
    <normaloff>:/img/img/icon.ico</normaloff>:/img/img/icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDateEdit" name="leClender">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>57</y>
     <width>151</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="ldate">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ldevicecode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>131</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>SpuerCode</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSuperPwd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>128</y>
     <width>151</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetSuperPwd">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>120</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>GetCode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>120</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QSuperPassword.qrc"/>
 </resources>
 <connections/>
</ui>
